lessons/12-css-pseudo-classes/index.html

```diff
@@ -0,0 +1,41 @@
+<html>
+    <head>
+       <title>Pseudo Classes</title>
+       <style>
+           div li:nth-child(2) :hover {
+               background: red;
+               color: white;
+           }
+
+           div li:first-child{
+               font-size: 32px;
+           }
+
+           div li:nth-child(3){
+               color: blue;
+           }
```

Comment:
Close! We want _every_ 3rd item to have this style, this would **only** target the 3rd element in the list

To do every 3rd item, we can place an `n` after the 3 like so:

```css
div li:nth-child(3n) {
  color: blue;
}
```